### PR TITLE
[15.0][MIG] base_kanban_stage

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
This module provides a stage model compatible with Kanban views and the
standard views needed to manage these stages. It also provides the
``base.kanban.abstract`` model, which can be inherited to add support for
Kanban views with stages to any other model. Lastly, it includes a base Kanban
view that can be extended as needed.

cc @ForgeFlow

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM

File: base_kanban_stage/README.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+* Odoo Community Association: `Icon <https://odoo-community.org/logo.png>`_.
+
+Contributors
+------------
+
+* Dave Lasley <dave@laslabs.com>
+* Oleg Bulkin <obulkin@laslabs.com>
+* Daniel Reis <dreis.pt@hotmail.com>
+* Alex Comba <alex.comba@agilebg.com>
+* Miquel Raïch <miquel.raich@eficent.com>
```
Comment (MiquelRForgeFlow) [inline base_kanban_stage/README.rst:96]:
```suggestion
* Miquel Raïch <miquel.raich@forgeflow.com>
```

**Review by JoanSForgeFlow — APPROVED** [review]
looks great! :)

File: base_kanban_stage/tests/test_base_kanban_abstract.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        model._auto_init()
+        model.init()
+        self.env["ir.model"]._reflect_models([model._name])
+        self.env["ir.model.fields"]._reflect_fields([model._name])
+        self.env["ir.model.fields.selection"]._reflect_selections([model._name])
+        self.env["ir.model.constraint"]._reflect_constraints([model._name])
+        while self.registry._post_init_queue:
+            func = self.registry._post_init_queue.popleft()
+            func()
+        return model
```
Comment (MiquelRForgeFlow) [inline base_kanban_stage/tests/test_base_kanban_abstract.py:38]:
```suggestion
        del self.registry._post_init_queue
        del self.registry._foreign_keys
        return model
```

File: base_kanban_stage/views/base_kanban_abstract.xml [hunk 3]
```diff
@@ -9,7 +9,6 @@
         <field name="model">base.kanban.abstract</field>
         <field name="arch" type="xml">
             <kanban default_group_by="stage_id" class="o_kanban_small_column">
-                <field name="display_name" />
```
Comment (MiquelRForgeFlow) [inline base_kanban_stage/views/base_kanban_abstract.xml]: why? shouldn't any substitute be valid, like "name" or similar?

**Pull request merged**